src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Resources/Theme/HCWhite.xaml

```diff
@@ -564,110 +564,110 @@
+    <Color x:Key="SurfaceStrokeColorInverse">#3D3D3D</Color>
 
-    <Color x:Key="DividerStrokeColorDefault">#FF0000</Color>
+    <Color x:Key="DividerStrokeColorDefault">#3D3D3D</Color>
 
-    <Color x:Key="FocusStrokeColorOuter">#FF0000</Color>
-    <Color x:Key="FocusStrokeColorInner">#FF0000</Color>
+    <Color x:Key="FocusStrokeColorOuter">#3D3D3D</Color>
+    <Color x:Key="FocusStrokeColorInner">#FFFAEF</Color>
 
-    <Color x:Key="CardBackgroundFillColorDefault">#FF0000</Color>
-    <Color x:Key="CardBackgroundFillColorSecondary">#FF0000</Color>
+    <Color x:Key="CardBackgroundFillColorDefault">#FFFAEF</Color>
+    <Color x:Key="CardBackgroundFillColorSecondary">#FFFAEF</Color>
 
-    <Color x:Key="SmokeFillColorDefault">#FF0000</Color>
+    <Color x:Key="SmokeFillColorDefault">#FFFAEF</Color>
 
-    <Color x:Key="LayerFillColorDefault">#FF0000</Color>
-    <Color x:Key="LayerFillColorAlt">#FF0000</Color>
-    <Color x:Key="LayerOnAcrylicFillColorDefault">#FF0000</Color>
-    <Color x:Key="LayerOnAccentAcrylicFillColorDefault">#FF0000</Color>
+    <Color x:Key="LayerFillColorDefault">#FFFAEF</Color>
+    <Color x:Key="LayerFillColorAlt">#FFFAEF</Color>
+    <Color x:Key="LayerOnAcrylicFillColorDefault">#FFFAEF</Color>
+    <Color x:Key="LayerOnAccentAcrylicFillColorDefault">#FFFAEF</Color>
 
     <!--  Fallback color for missing Acrylic used for e.g. Flyouts  -->
-    <Color x:Key="AcrylicBackgroundFillColorDefault">#FF0000</Color>
+    <Color x:Key="AcrylicBackgroundFillColorDefault">#FFFAEF</Color>
 
     <Color x:Key="LayerOnMicaBaseAltFillColorDefault">#FF0000</Color>
     <Color x:Key="LayerOnMicaBaseAltFillColorSecondary">#FF0000</Color>
     <Color x:Key="LayerOnMicaBaseAltFillColorTertiary">#FF0000</Color>
     <Color x:Key="LayerOnMicaBaseAltFillColorTransparent">#FF0000</Color>
 
-    <Color x:Key="SolidBackgroundFillColorBase">#FF0000</Color>
-    <Color x:Key="SolidBackgroundFillColorSecondary">#FF0000</Color>
-    <Color x:Key="SolidBackgroundFillColorTertiary">#FF0000</Color>
-    <Color x:Key="SolidBackgroundFillColorQuarternary">#FF0000</Color>
+    <Color x:Key="SolidBackgroundFillColorBase">#FFFAEF</Color>
+    <Color x:Key="SolidBackgroundFillColorSecondary">#FFFAEF</Color>
+    <Color x:Key="SolidBackgroundFillColorTertiary">#FFFAEF</Color>
+    <Color x:Key="SolidBackgroundFillColorQuarternary">#FFFAEF</Color>
     <Color x:Key="SolidBackgroundFillColorTransparent">#FF0000</Color>
     <Color x:Key="SolidBackgroundFillColorBaseAlt">#FF0000</Color>
 
-    <Color x:Key="SystemFillColorSuccess">#FF0000</Color>
-    <Color x:Key="SystemFillColorCaution">#FF0000</Color>
-    <Color x:Key="SystemFillColorCritical">#FF0000</Color>
-    <Color x:Key="SystemFillColorNeutral">#FF0000</Color>
+    <Color x:Key="SystemFillColorSuccess">#3D3D3D</Color>
+    <Color x:Key="SystemFillColorCaution">#3D3D3D</Color>
+    <Color x:Key="SystemFillColorCritical">#F3D3D3DF0000</Color>
```

Comment:
The Color here is not a valid windows color. I suppose the required color here is #F3D3D3DF.

Reply:
Corrected, thank you